decidim-proposals/spec/system/admin/highlighted_proposal_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Highlighted proposal", type: :system do
+  let(:admin) { create :user, :admin, :confirmed }
+  let(:organization) { admin.organization }
+  let!(:participatory_process) { create(:participatory_process, organization: organization) }
+  let!(:proposal_component) { create(:proposal_component, participatory_space: participatory_process) }
+  let!(:proposals) do
+    [
+      create(:proposal, :not_answered, id: 1, created_at: 9.days.ago, published_at: 9.days.ago, component:
+        proposal_component),
+      create(:proposal, :not_answered, id: 2, created_at: 5.days.ago, published_at: 5.days.ago, component:
+        proposal_component),
+      create(:proposal, :with_answer, id: 3, created_at: 9.days.ago, published_at: 9.days.ago, component:
+        proposal_component)
+    ]
+  end
+
```

Comment:
```suggestion
let(:days_overdue) { 7 }
```
This var will be used to allow us define different values for the configuration value  `:unanswered_proposals_overdue` config.